website/docs/hardware/antenna/testing.md

```diff
@@ -22,7 +22,7 @@ On the topic of testing - performing your own testing and providing feedback is
 
 ## Signal strength testing
 
-Real world testing is also discussed by Andreas Speiss (the 'guy with the Swiss accent') in his [tutorial](https://www.youtube.com/watch?v=J3PBL9oLPX8). He has written [code](https://github.com/SensorsIot/Antenna-Tester) for testing antennas using two Lora32 V1 boards to compare how different antennas behave.
+Real world testing is also discussed by Andreas Speiss (the 'guy with the Swiss accent') in his [tutorial](https://www.youtube.com/watch?v=J3PBL9oLPX8). He has written [code](https://github.com/SensorsIot/Antenna-Tester) for testing antennas using two Lora32 V1 boards to compare how different antennas behave. Lilygo have also made code available for testing the rssi on the [LORA32](https://github.com/LilyGO/TTGO-LORA32) [boards](https://github.com/Xinyuan-LilyGO/TTGO-LoRa-Series) and the [T-Beam](https://github.com/LilyGO/TTGO-T-Beam).
```

Comment:
I'd forgotten all about this, I'm going to need to check it out!

Reply:
I got a pair of TTGO Lora v1.0 boards just for this purpose! Works fairly well, just need to set up a permanent testing rig.